R/RLearner_regr_cforest.R

```diff
@@ -59,3 +59,14 @@ trainLearner.regr.cforest = function(.learner, .task, .subset, .weights = NULL,
 predictLearner.regr.cforest = function(.learner, .model, .newdata, ...) {
   as.vector(predict(.model$learner.model, newdata = .newdata, ...))
 }
+
+getFeatureImportance.regr.cforest = function(.learner, .model, AUC = FALSE, ...) {
```

Comment:
The AUC option is not needed for regression.
What I found puzzling at first is that I could call `varimpAUC` for a regression forest without getting an error or a warning.
But when you look at the source code of `varimp` and `varimpAUC` you see that the AUC option is only relevant for binary regression and that both functions do exactly the same in the regression case (decrease in MSE).